<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawConfirmWidget</class>
 <widget class="QWidget" name="WithdrawConfirmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>189</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(198, 202, 212);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>提至地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>交易确认</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_address">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>215</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(51, 51, 51);
font: 14px &quot;Microsoft YaHei UI Light&quot;;}</string>
   </property>
   <property name="text">
    <string>address</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>259</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(198, 202, 212);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>提现数额</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>259</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(198, 202, 212);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>提现手续费</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>319</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(198, 202, 212);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>共需支付</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>379</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(198, 202, 212);
font: 12px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>交易密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>400</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_actualNumber">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>285</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 51, 51);
font: 13px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_feeNumber">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>285</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 51, 51);
font: 13px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalNumber">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>345</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(84,116,235);
font: 13px &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_confirm">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>449</y>
     <width>96</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_cancel">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>449</y>
     <width>96</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
